src/components/ui/login/index.tsx

```
@@ -57,40 +57,39 @@ export default function Login() {
-
-			localStorage.setItem('AT', headerToken);
-
+			
 			if (!user) {
 				throw new Error('로그인 유저 정보가 없습니다.');
 			}
 			setAuth(user);
+			setToken(headerToken)
 			alertStore.success('로그인 성공');
 			loginModal.close();
 		} catch (e: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)